doc/source/index.rst

```diff
@@ -2,7 +2,8 @@
 GDAL
 ================================================================================
 
-.. include:: ./about_no_title.rst
+.. include:: ./about.rst
+   :start-line: 6
```

Comment:
@rouault what is your feeling about using [:start-after:](https://docutils.sourceforge.io/docs/ref/rst/directives.html#include) with a developer comment we would add in the about file, instead of line number?

```suggestion
   :start-after: .. descriptionstartshere
```

Reply:
sounds great!